src/public/index.html

```diff
@@ -9,6 +9,10 @@
                     navigator.serviceWorker.register('/sw.js');
                 });
             }
+            const actionName = {
```

Comment:
Can we just define runtime global variable on window? Why to trash local-storage?